GOVERNANCE.md

```diff
@@ -0,0 +1,143 @@
+### Build Working Group
+
+The Node.js Build Working Group is jointly governed by a Working Group (WG)
```

Comment:
where was this copied from? "joint" would need to list multiple owners. This whole paragraph seems like its from some joint commcomm/node repo.

Reply:
In the description I link to where I got the original from, its from the TSC's bootstrap governance doc and is what the release doc is based on. I think jointly is in referral to each member of the group having a say in the governance of the working group.

Reply:
I don't think lines 3-13 make sense for build wg.  https://github.com/nodejs/build#purpose is a better description